packages/client/hmi-client/src/services/toast.ts

```diff
@@ -0,0 +1,95 @@
+import ToastEventBus from 'primevue/toasteventbus';
+
+const DEFAULT_DURATION = 3000; // 3 seconds
+
+// TODO: Define more.
+export enum ToastSummaries {
+	NETWORK_ERROR = 'Network Error',
+	UNKNOWN_ERROR = 'Unknown Error',
+	SERVICE_UNAVAILABLE = 'Service Unavailable',
+	ATTENTION = 'Attention',
+	INFO = 'Info',
+	WARNING = 'Warning',
+	ERROR = 'Error'
+}
+
+export enum ToastSeverity {
+	info = 'info',
+	warn = 'warn',
+	success = 'success',
+	error = 'error'
+}
+
+export const useToastService = () => {
+	const showToast = (
+		severity: ToastSeverity,
+		summary: string,
+		detail: string,
+		life: number = 4000
+	) => {
+		ToastEventBus.emit('add', {
+			severity: severity,
```

Comment:
In JS now you don't need to explicitely write the value if it's the same name as the property.
i.e. `{ severety, summary, detail, life }` will do it.